andela/technites-bn-backend
# #168050016 Delete comment

#### What does this PR do?
 - allow a user to delete a comment so that, the user can ensure that they are conveying the right information

#### Description of Task to be completed?
 - add delete feature controller 
#### How should this be manually tested?
 - run `git fetch --all` to get all branches'
 - run `npm run db:reset` to migrate and seed the db
 - run server
 - create a request
 - comment on the request post `/api/v1/requests/${requestId}/comments'
 - delete comment using `delete /api/v1/requests/${requestId}/comments/${commentId}`
 - confirm deletion by getting comments from a specific request with get`/api/v1/requests/${requestId}/comments`
 - cehckout to 't-delete-comment-168050016'
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
[#168050016](https://www.pivotaltracker.com/story/show/168050016)
#### What are the relevant Github Issues?
#### Screenshots (if appropriate)
#### Questions:
